.github/workflows/build.yml

```diff
@@ -10,8 +10,8 @@ jobs:
     strategy:
       matrix:
         os: ["ubuntu-20.04"]
-        elixir: ["1.13"]
-        otp: ["24"]
+        elixir: ["1.16"]
```

Comment:
This is just the lint step. It may find that the formatting is correct for 1.14 but not for 1.16. Generally speaking, for libraries I believe if we just lint with the latest version, it should be backwards compatible.

Reply:
Whoops, totally missed that 😄